internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -0,0 +1,338 @@
+			gatewayKey := resolveBTPolicyTargetRef(policy, gatewayMap, routeMap)
+			if gatewayKey == nil {
+				continue
+			}
+
+			// Check if another policy targeting the same section exists
+			section := string(*(policy.Spec.TargetRef.SectionName))
+			s, ok := targetedGateways[*gatewayKey]
+			if ok && s.Has(section) {
+				message := "Unable to target section, another BackendTrafficPolicy has already attached to it"
+
+				status.SetBackendTrafficPolicyCondition(policy,
+					gwv1a2.PolicyConditionAccepted,
+					metav1.ConditionFalse,
+					gwv1a2.PolicyReasonConflicted,
+					message,
+				)
+				continue
+			}
+
+			// Check if any policies that target xRoute resources are overriding this policy so we can set the status
+			if _, ok := overriddenPolicyTargets[*gatewayKey]; ok {
+				message := "There are existing BackendTrafficPolicies that are overriding this config at the route level"
+
+				status.SetBackendTrafficPolicyCondition(policy,
+					egv1a1.PolicyConditionOverridden,
+					metav1.ConditionTrue,
+					egv1a1.PolicyReasonOverridden,
+					message,
+				)
+			}
+
+			// Add section to targeted gateways map
+			if s == nil {
+				targetedGateways[*gatewayKey] = sets.New[string]()
+			}
+			targetedGateways[*gatewayKey].Insert(section)
+
+			translateBackendTrafficPolicy(policy, xdsIR)
+
+			// Set Accepted=True
+			status.SetBackendTrafficPolicyCondition(policy,
+				gwv1a2.PolicyConditionAccepted,
+				metav1.ConditionTrue,
+				gwv1a2.PolicyReasonAccepted,
+				"BackendTrafficPolicy has been accepted.",
+			)
+		}
+	}
+
+	// Process the policies targeting Gateways with no section name set
+	for _, policy := range backendTrafficPolicies {
+		if policy.Spec.TargetRef.SectionName == nil && policy.Spec.TargetRef.Kind == KindGateway {
+
+			policy := policy.DeepCopy()
+			res = append(res, policy)
+
+			// Negative statuses have already been assigned so its safe to skip
+			gatewayKey := resolveBTPolicyTargetRef(policy, gatewayMap, routeMap)
+			if gatewayKey == nil {
+				continue
+			}
+
+			// Check if another policy targeting the same Gateway exists
+			s, ok := targetedGateways[*gatewayKey]
+			if ok && s.Has(AllSections) {
+				message := "Unable to target Gateway, another BackendTrafficPolicy has already attached to it"
+
+				status.SetBackendTrafficPolicyCondition(policy,
+					gwv1a2.PolicyConditionAccepted,
+					metav1.ConditionFalse,
+					gwv1a2.PolicyReasonConflicted,
+					message,
+				)
+
+				continue
+
+			}
+
+			if ok && (s.Len() > 0) {
+				// Maintain order here to ensure status/string does not change with same data
+				sections := s.UnsortedList()
+				sort.Strings(sections)
+				message := fmt.Sprintf("There are existing BackendTrafficPolicies that are overriding these sections %v", sections)
+
+				status.SetBackendTrafficPolicyCondition(policy,
+					egv1a1.PolicyConditionOverridden,
+					metav1.ConditionTrue,
+					egv1a1.PolicyReasonOverridden,
+					message,
+				)
+			}
+
```

Comment:
We should also check if there are existing policies at the route level.

Reply:
I believe this is happening on L109

Reply:
L109 `overriddenPolicyTargets` is only used to check conflicts with listeners at L152. Ideally, we should also check if there are existing policies at the route level that override the gateway-level policies.